### DISCUSSION TITLE
Check for updates in Admin page gives error

### DISCUSSION THREAD

**Discussion by alexanderdelargeNL** [firefly-iii/firefly-iii#9560] [Firefly III]
When I press the button for checking for updates, I receive the error below. How can I fix this? I'm running Firefly III 6.1.24 locally on my Windows 10. 

`Guzzle: cURL error 60: SSL certificate problem: unable to get local issuer certificate (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://version.firefly-iii.org/index.json`

**Comment by JC5** [comment]
Can you browse to the site manually without issue?

**Comment by alexanderdelargeNL** [comment]
Yes, in the same browser window I can browse to https://version.firefly-iii.org/index.json without issue

**Reply by JC5** [reply]
You're not using Docker or anything, but like Apache for Windows?

**Reply by alexanderdelargeNL** [reply]
Correct, no Docker. Just Windows 10 x64 and Wampserver64

**Reply by JC5** [reply]
Mmmm, check if your Apache has a reference to certificate files in the directory somewhere. Something like "certificates.pem", idk what it's called on Windows.

If it's a little outdated, the root certificates may need to be updated.

**Reply by alexanderdelargeNL** [reply]
Issue is resolved. You were right, the issue was indeed related to certificates. For reason unknown to my the Apache in my WAMP setup could not verify the (CA) certificates via cURL. 

I downloaded the latest cacert.pem from the cURL website and modified php.ini accordingly. Did a restart of Apache and that was it. I couldn't have solved this without your suggestions (and ChatGPT's), so thank you James!

**Reply by JC5** [reply]
Glad to hear it's fixed!
